src/System.Runtime.Extensions/tests/System/Environment.Exit.cs

```diff
@@ -42,6 +42,15 @@ public static void CheckExitCode(int expectedExitCode)
 
         [Theory]
         [MemberData(nameof(ExitCodeValues))]
+        [SkipOnTargetFramework(~TargetFrameworkMonikers.UapNotUapAot, "RemoteExecutor behaves different in UAP.")]
+        public static void CheckExitCode_Uap(int expectedExitCode)
+        {
+            // Shouldn't fail as RemoteExecutor verifies that the exit code is the same as Options.ExpectedExitCode
+            RemoteInvoke(s => int.Parse(s), expectedExitCode.ToString(), new RemoteInvokeOptions { ExpectedExitCode = expectedExitCode });
```

Comment:
RemoteInvokeHandle.Dispose() the only thing that does is checks for the process's exit code and kills the process. In UAP the process inside the handle is null, if the process is null it doesn't do anything. Should I still add .Dispose()?